src/views/ops/Adhoc/QuickJob.vue

```diff
@@ -157,10 +157,6 @@ export default {
                   assets: hosts,
                   query: query
                 }).then(data => {
-                  if (Array.isArray(data) && data.length === 0) {
-                    this.$message.info(`${this.$t('NoAccountFound')}`)
-                    return cb([])
-                  }
                   const ns = data.map(item => {
                     return { value: item.username }
                   })
```

Comment:
There does not appear to be any irregularities, potential issues, or optimizations needed specific to this line of code. The current code functionally works as it currently is with the provided arguments and options for exporting hostnames from an API response.